misc/multiaddr/src/storage.rs

```diff
@@ -0,0 +1,71 @@
+use std::sync::Arc;
+
+/// MAX_INLINE is the maximum size of a multiaddr that can be stored inline
+const MAX_INLINE: usize = 38;
```

Comment:
This means that the entire struct will be 40 bytes large (+1 byte enum discriminator and +1 byte length). I guess you could argue that for the typical multiaddr that does not contain a peer id, 32 bytes total length / 30 bytes payload length would be enough...

Reply:
I would say having these assumptions documented similar to as  comments similar to https://github.com/multiformats/rust-multihash/blob/master/src/storage.rs#L5 would be useful for the future.

Reply:
Will push a documentation update!